src/tokens/solana.tokenlist.json

```diff
@@ -11733,6 +11733,19 @@
         "twitter": "https://twitter.com/mangomarkets",
         "discord": "https://discord.gg/67jySBhxrg"
       }
+    },
+    {
+      "chainId": 101,
+      "address": "7dHbWXmci3dT8UFYWYZweBLXgycu7Y3iL6trKn1Y7ARj",
+      "symbol": "stSOL",
+      "name": "Lido Staked SOL",
+      "decimals": 9,
+      "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/7dHbWXmci3dT8UFYWYZweBLXgycu7Y3iL6trKn1Y7ARj/logo.png",
+      "tags": [],
```

Comment:
We can set `"tags": ["lp-token"]`: https://github.com/solana-labs/token-list/blob/fa930e633feaec0b7d7672ca67b2da0785803511/src/tokens/solana.tokenlist.json#L17-L20